charts/tfy-clickhouse/values.yaml

```diff
@@ -58,8 +61,8 @@ zookeeper:
   enabled: true
   replicaCount: 1
   image:
-    registry: "tfy.jfrog.io/tfy-mirror"
-    repository: bitnami/zookeeper
+    registry: "tfy.jfrog.io"
```

Comment:
```suggestion
    registry: ""
```

Reply:
for this we require it here. 
zookeeper doesnt rely on. global.image.registry